### PR TITLE
Enabled device_type AUTO:GPU.1,CPU in OVEP

### PR DESCRIPTION
**Pull request by ankitm3k**
### Description
This PR adds new options to enable devices during inference runtime for machines having a iGPU (GPU.0) and dGPU (GPU.1)

### Motivation and Context
We are now able to run inference with device_type as AUTO:GPU.1,CPU and further more combinations
- This PR fixes the issue for ticket: https://jira.devtools.intel.com/browse/HAFP-2794

### CODE REVIEW

File: onnxruntime/core/providers/openvino/openvino_execution_provider.h [hunk 1]
```diff
@@ -16,16 +16,23 @@
 
 namespace onnxruntime {
 
+struct OVDevices {
+  ov::Core core;
+  std::vector<std::string> ovDevices() const {
```
Comment (sfatimar) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.h:21]: The naming convention does not seem right

Comment (sfatimar) [reply onnxruntime/core/providers/openvino/openvino_execution_provider.h:21]: it should get_ov_devices()

**Comment by sfatimar** [comment]
Please close this PR if it is not rejected

**Comment by sfatimar** [comment]
Please rebase the branch .... the PR shows multiple commits which are not part of the fix

File: onnxruntime/core/providers/openvino/openvino_execution_provider.h (hunk 1 above)
Comment (ankitm3k) [reply onnxruntime/core/providers/openvino/openvino_execution_provider.h]: fixed

**Comment by ankitm3k** [comment]
Rebased the changes

**Comment by sfatimar** [comment]
Not going to merge

**Pull request closed without merging**